index.js

```diff
@@ -2277,6 +2278,35 @@
       [$.FiniteNumber, $.FiniteNumber],
       function(a) { return a - 1; });
 
+  //# mult :: FiniteNumber -> FiniteNumber -> FiniteNumber
+  //.
+  //. Returns the product of two (finite) numbers.
+  //.
+  //. ```javascript
+  //. > S.mult(4, 2)
+  //. 8
+  //. ```
+  S.mult =
+  def('mult',
+      {},
+      [$.FiniteNumber, $.FiniteNumber, $.FiniteNumber],
+      function(a, b) { return a * b; });
+
+  //# div :: FiniteNumber -> NonZeroFiniteNumber -> FiniteNumber
+  //.
+  //. Returns the result of dividing its first argument (a finite number) by
+  //. its second argument (a non-zero finite number).
+  //.
+  //. ```javascript
+  //. > S.div(4, 2)
```

Comment:
Fair enough, although regarding making it clear that this isn't integer division, I'm not really sure how much clearer we can make it given the type signature at the top of the function description.